ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/transform/PubsubMessageToObjectNode.java

```diff
@@ -330,8 +330,12 @@ private void processField(String jsonFieldName, Field field, JsonNode value, Obj
         ObjectNode additionalProperties) {
       final String bqFieldName = field.getName();
 
-      // A record of key and value indicates we need to transformForBqSchema a map to an array.
-      if (isMapType(field)) {
+      // null is valid for any type except an element of a list
+      if (value.isNull()) {
+        updateParent(parent, jsonFieldName, bqFieldName, value);
+
+        // A record of key and value indicates we need to transformForBqSchema a map to an array.
+      } else if (isMapType(field)) {
         expandMapType(jsonFieldName, (ObjectNode) value, field, parent, additionalProperties);
```

Comment:
the exception in #1317 is thrown by this line trying to cast a literal `null` value to `ObjectNode` here.

This PR fixes that by allowing `null` for any `value` passed to `processField`, which I believe is called on every type that is not an element of a list.

The resulting behavior omits the field, which BigQuery will interpret as an empty list if necessary.